functions/src/types/UserData.ts

```diff
@@ -4,8 +4,10 @@ import { UID } from '../types/uid'
 
 export interface UserData {
     id: UID
-    subscriptionActive: boolean
-    quotaReached: boolean
+    subscription: {
```

Comment:
no more need of `history` and `type` fields ?

Reply:
it's not used currently, and not populated automatically so yeah.